src/v/cluster/metadata_cache.cc

```diff
@@ -217,6 +217,10 @@ std::optional<model::node_id> metadata_cache::get_controller_leader_id() {
 
 void metadata_cache::reset_leaders() { _leaders.local().reset(); }
 
+ss::future<> metadata_cache::refresh_health_monitor() {
+    co_await _health_monitor.local().refresh_info();
```

Comment:
the timer in the health monitor frontend runs on the `refresher_shard`. yet here it looks like we invoke the same functionality that that timer triggers from any shard. should it also run on the same shard?